content/nginx-one-console/nginx-configs/certificates/csg-unmanaged-certificates.md

```diff
@@ -0,0 +1,115 @@
+---
+nd-content-type: concept
+nd-docs: DOCS-000
+nd-product: NONECO
+title: Unmanaged certificates in Config Sync Groups
+toc: true
+weight: 200
+---
+
+## Overview
+
+Unmanaged certificates are SSL/TLS certificates that you install and manage manually on NGINX instances. Unlike managed certificates that are uploaded and distributed through the NGINX One Console, unmanaged certificates are installed directly on individual instances and referenced by their file paths in NGINX configuration files. You are responsible for distributing, updating, and maintaining these certificates across your infrastructure.
+
+### Unmanaged certificates in Config Sync Groups
+Config Sync Groups (CSGs) in NGINX One Console ensure configuration consistency across connected NGINX instances. While managed certificates uploaded through the Console are automatically synchronized and tracked, unmanaged certificates follow a different model that provides visibility without automated management.
+
+When you use unmanaged certificates in a CSG, NGINX One Console does not synchronize the certificate files themselves. However, it tracks their metadata to help you verify consistency across instances and understand the state of your certificates.
+
+## How unmanaged certificates work in Config Sync Groups
+
+### Certificate tracking and visibility
+
+When you use unmanaged certificates in a Config Sync Group:
+
+- The NGINX Agent collects certificate metadata from each instance
+- The Console displays unmanaged certificates based on their file paths and metadata
+- Certificate consistency is determined by comparing certificate contents and file paths across instances
+
+### Consistent certificates
+
+When all instances in a CSG reference identical certificate files with the same file paths:
+
+- Their contents and metadata match across all instances
+- The CSG displays a single unmanaged certificate entry for that file path
+
+### Inconsistent certificates
+
+If certificate contents differ between instances, even when file paths are the same:
+
+- Each unique certificate appears as a separate unmanaged entry in the Console
+- Certificates are identified by their content and associated instance
+- The CSG displays separate certificate entries in the configuration
+
+If certificate file paths differ between instances:
+
+- CSG publication may fail
```

Comment:
Just checking. Is it CSG or certificate publication that can fail?

Reply:
If it is CSG publication that can fail, I'd change current line 44 (comment added there)